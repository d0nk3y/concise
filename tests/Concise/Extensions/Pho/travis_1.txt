Concise by Elliot Chance.

0 assertions, 0 seconds                                                                                                                 0 / 110 (  0%)
0 assertions, 0 seconds                                                                                                                 2 / 110 (  1%)
0 assertions, 0 seconds                                                                                                                 3 / 110 (  2%)
0 assertions, 0 seconds                                                                                                                 4 / 110 (  3%)
0 assertions, 0 seconds                                                                                                                 5 / 110 (  4%)
0 assertions, 0 seconds                                                                                                                 6 / 110 (  5%)
0 assertions, 0 seconds                                                                                                                 7 / 110 (  6%)
0 assertions, 0 seconds                                                                                                                 8 / 110 (  7%)
0 assertions, 0 seconds                                                                                                                 9 / 110 (  8%)
0 assertions, 0 seconds                                                                                                                10 / 110 (  9%)
0 assertions, 0 seconds                                                                                                                11 / 110 ( 10%)
0 assertions, 0 seconds                                                                                                                13 / 110 ( 11%)
0 assertions, 0 seconds                                                                                                                14 / 110 ( 12%)
0 assertions, 0 seconds                                                                                                                15 / 110 ( 13%)
0 assertions, 0 seconds                                                                                                                16 / 110 ( 14%)
0 assertions, 0 seconds                                                                                                                17 / 110 ( 15%)
0 assertions, 0 seconds                                                                                                                18 / 110 ( 16%)
0 assertions, 0 seconds                                                                                                                19 / 110 ( 17%)
0 assertions, 0 seconds                                                                                                                20 / 110 ( 18%)
0 assertions, 0 seconds                                                                                                                21 / 110 ( 19%)
0 assertions, 0 seconds                                                                                                                22 / 110 ( 20%)
0 assertions, 0 seconds                                                                                                                24 / 110 ( 21%)
0 assertions, 0 seconds                                                                                                                25 / 110 ( 22%)
0 assertions, 0 seconds                                                                                                                26 / 110 ( 23%)
0 assertions, 0 seconds                                                                                                                27 / 110 ( 24%)
0 assertions, 0 seconds                                                                                                                28 / 110 ( 25%)
0 assertions, 0 seconds                                                                                                                29 / 110 ( 26%)
0 assertions, 0 seconds                                                                                                                30 / 110 ( 27%)
0 assertions, 0 seconds                                                                                                                31 / 110 ( 28%)
0 assertions, 0 seconds                                                                                                                32 / 110 ( 29%)
0 assertions, 0 seconds                                                                                                                33 / 110 ( 30%)
0 assertions, 0 seconds                                                                                                                35 / 110 ( 31%)
0 assertions, 0 seconds                                                                                                                36 / 110 ( 32%)
0 assertions, 0 seconds                                                                                                                37 / 110 ( 33%)
0 assertions, 0 seconds                                                                                                                38 / 110 ( 34%)
0 assertions, 0 seconds                                                                                                                39 / 110 ( 35%)
0 assertions, 0 seconds                                                                                                                40 / 110 ( 36%)
0 assertions, 0 seconds                                                                                                                41 / 110 ( 37%)
0 assertions, 0 seconds                                                                                                                42 / 110 ( 38%)
0 assertions, 0 seconds                                                                                                                43 / 110 ( 39%)
0 assertions, 0 seconds                                                                                                                44 / 110 ( 40%)
0 assertions, 0 seconds                                                                                                                46 / 110 ( 41%)
0 assertions, 0 seconds                                                                                                                47 / 110 ( 42%)
0 assertions, 0 seconds                                                                                                                48 / 110 ( 43%)
0 assertions, 0 seconds                                                                                                                49 / 110 ( 44%)
0 assertions, 0 seconds                                                                                                                50 / 110 ( 45%)
0 assertions, 0 seconds                                                                                                                51 / 110 ( 46%)
0 assertions, 0 seconds                                                                                                                52 / 110 ( 47%)
0 assertions, 0 seconds                                                                                                                53 / 110 ( 48%)
0 assertions, 0 seconds                                                                                                                54 / 110 ( 49%)
0 assertions, 0 seconds                                                                                                                55 / 110 ( 50%)
0 assertions, 0 seconds                                                                                                                57 / 110 ( 51%)
0 assertions, 0 seconds                                                                                                                58 / 110 ( 52%)
0 assertions, 0 seconds                                                                                                                59 / 110 ( 53%)
0 assertions, 0 seconds                                                                                                                60 / 110 ( 54%)
0 assertions, 0 seconds                                                                                                                61 / 110 ( 55%)
0 assertions, 0 seconds                                                                                                                62 / 110 ( 56%)
0 assertions, 0 seconds                                                                                                                63 / 110 ( 57%)
0 assertions, 0 seconds                                                                                                                64 / 110 ( 58%)
0 assertions, 0 seconds                                                                                                                65 / 110 ( 59%)
0 assertions, 0 seconds                                                                                                                66 / 110 ( 60%)
0 assertions, 0 seconds                                                                                                                68 / 110 ( 61%)
0 assertions, 0 seconds                                                                                                                69 / 110 ( 62%)
0 assertions, 0 seconds                                                                                                                70 / 110 ( 63%)
0 assertions, 0 seconds                                                                                                                71 / 110 ( 64%)
0 assertions, 0 seconds                                                                                                                72 / 110 ( 65%)
0 assertions, 0 seconds                                                                                                                73 / 110 ( 66%)
0 assertions, 0 seconds                                                                                                                74 / 110 ( 67%)
0 assertions, 0 seconds                                                                                                                75 / 110 ( 68%)
0 assertions, 0 seconds                                                                                                                76 / 110 ( 69%)
0 assertions, 0 seconds                                                                                                                77 / 110 ( 70%)
0 assertions, 0 seconds                                                                                                                79 / 110 ( 71%)
0 assertions, 0 seconds                                                                                                                80 / 110 ( 72%)
0 assertions, 0 seconds                                                                                                                81 / 110 ( 73%)
0 assertions, 0 seconds                                                                                                                82 / 110 ( 74%)
0 assertions, 0 seconds                                                                                                                83 / 110 ( 75%)
0 assertions, 0 seconds                                                                                                                84 / 110 ( 76%)
0 assertions, 0 seconds                                                                                                                85 / 110 ( 77%)
0 assertions, 0 seconds                                                                                                                86 / 110 ( 78%)
0 assertions, 0 seconds                                                                                                                87 / 110 ( 79%)
0 assertions, 0 seconds                                                                                                                88 / 110 ( 80%)
0 assertions, 0 seconds                                                                                                                90 / 110 ( 81%)
0 assertions, 0 seconds                                                                                                                91 / 110 ( 82%)
0 assertions, 0 seconds                                                                                                                92 / 110 ( 83%)
0 assertions, 0 seconds                                                                                                                93 / 110 ( 84%)
0 assertions, 0 seconds                                                                                                                94 / 110 ( 85%)
0 assertions, 0 seconds                                                                                                                95 / 110 ( 86%)
0 assertions, 0 seconds                                                                                                                96 / 110 ( 87%)
0 assertions, 0 seconds                                                                                                                97 / 110 ( 88%)
0 assertions, 0 seconds                                                                                                                98 / 110 ( 89%)
0 assertions, 0 seconds                                                                                                                99 / 110 ( 90%)
0 assertions, 0 seconds                                                                                                               101 / 110 ( 91%)
0 assertions, 0 seconds                                                                                                               102 / 110 ( 92%)
0 assertions, 0 seconds                                                                                                               103 / 110 ( 93%)
0 assertions, 0 seconds                                                                                                               104 / 110 ( 94%)
0 assertions, 0 seconds                                                                                                               105 / 110 ( 95%)
0 assertions, 0 seconds                                                                                                               106 / 110 ( 96%)
0 assertions, 0 seconds                                                                                                               107 / 110 ( 97%)
0 assertions, 0 seconds                                                                                                               108 / 110 ( 98%)
0 assertions, 0 seconds                                                                                                               109 / 110 ( 99%)
0 assertions, 0 seconds                                                                                                               110 / 110 (100%)
