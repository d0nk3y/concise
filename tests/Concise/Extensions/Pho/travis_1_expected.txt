

  0 / 110 (  0%)
  2 / 110 (  1%)
  3 / 110 (  2%)
  4 / 110 (  3%)
  5 / 110 (  4%)
  6 / 110 (  5%)
  7 / 110 (  6%)
  8 / 110 (  7%)
  9 / 110 (  8%)
 10 / 110 (  9%)
 11 / 110 ( 10%)
 13 / 110 ( 11%)
 14 / 110 ( 12%)
 15 / 110 ( 13%)
 16 / 110 ( 14%)
 17 / 110 ( 15%)
 18 / 110 ( 16%)
 19 / 110 ( 17%)
 20 / 110 ( 18%)
 21 / 110 ( 19%)
 22 / 110 ( 20%)
 24 / 110 ( 21%)
 25 / 110 ( 22%)
 26 / 110 ( 23%)
 27 / 110 ( 24%)
 28 / 110 ( 25%)
 29 / 110 ( 26%)
 30 / 110 ( 27%)
 31 / 110 ( 28%)
 32 / 110 ( 29%)
 33 / 110 ( 30%)
 35 / 110 ( 31%)
 36 / 110 ( 32%)
 37 / 110 ( 33%)
 38 / 110 ( 34%)
 39 / 110 ( 35%)
 40 / 110 ( 36%)
 41 / 110 ( 37%)
 42 / 110 ( 38%)
 43 / 110 ( 39%)
 44 / 110 ( 40%)
 46 / 110 ( 41%)
 47 / 110 ( 42%)
 48 / 110 ( 43%)
 49 / 110 ( 44%)
 50 / 110 ( 45%)
 51 / 110 ( 46%)
 52 / 110 ( 47%)
 53 / 110 ( 48%)
 54 / 110 ( 49%)
 55 / 110 ( 50%)
 57 / 110 ( 51%)
 58 / 110 ( 52%)
 59 / 110 ( 53%)
 60 / 110 ( 54%)
 61 / 110 ( 55%)
 62 / 110 ( 56%)
 63 / 110 ( 57%)
 64 / 110 ( 58%)
 65 / 110 ( 59%)
 66 / 110 ( 60%)
 68 / 110 ( 61%)
 69 / 110 ( 62%)
 70 / 110 ( 63%)
 71 / 110 ( 64%)
 72 / 110 ( 65%)
 73 / 110 ( 66%)
 74 / 110 ( 67%)
 75 / 110 ( 68%)
 76 / 110 ( 69%)
 77 / 110 ( 70%)
 79 / 110 ( 71%)
 80 / 110 ( 72%)
 81 / 110 ( 73%)
 82 / 110 ( 74%)
 83 / 110 ( 75%)
 84 / 110 ( 76%)
 85 / 110 ( 77%)
 86 / 110 ( 78%)
 87 / 110 ( 79%)
 88 / 110 ( 80%)
 90 / 110 ( 81%)
 91 / 110 ( 82%)
 92 / 110 ( 83%)
 93 / 110 ( 84%)
 94 / 110 ( 85%)
 95 / 110 ( 86%)
 96 / 110 ( 87%)
 97 / 110 ( 88%)
 98 / 110 ( 89%)
 99 / 110 ( 90%)
101 / 110 ( 91%)
102 / 110 ( 92%)
103 / 110 ( 93%)
104 / 110 ( 94%)
105 / 110 ( 95%)
106 / 110 ( 96%)
107 / 110 ( 97%)
108 / 110 ( 98%)
109 / 110 ( 99%)
110 / 110 (100%)
